cdap-docs/reference-manual/source/http-restful-api/artifact.rst

```diff
@@ -451,29 +450,23 @@ of the ``cdap-etl-batch`` artifact (pretty-printed and reformatted to fit):
     |$| GET <base-url>/namespaces/default/artifacts/cdap-etl-batch/versions/|release|/extensions/transform?scope=system
 
     [
-      {
-        "artifact": {
-          "name": "cdap-etl-lib",
-          "scope": "SYSTEM",
-          "version": "|release|-batch"
+        {
+            "name": "LogParser",
+            "type": "transform",
+            "description": "Parses logs from any input source for relevant information such as URI, IP, browser, device, HTTP status code, and timestamp.",
+            "className": "co.cask.hydrator.plugin.transform.LogParserTransform",
+            "artifact": {
+                "name": "core-plugins",
+                "version": "1.4.0-SNAPSHOT",
```

Comment:
I have brought in code I was working on in another branch that exposes a Cask Hydrator Version, and so have update these two examples to use it.